packages/server/controllers/guilds.js

```diff
@@ -82,6 +82,38 @@ async function getGuildMembers(guildId) {
   return members;
 }
 
+async function guildLeaderboard(guildId) {
+  const guildLeaderboard = await prisma.guildMembers.findMany({
+    where: {
+      guildId: guildId,
+    },
+
+    include: {
+      members: {
+        select: {
+          firstName: true,
+          lastName: true,
+          handler: true,
+          avatar: true,
+        },
+      },
+    },
+
+    orderBy: [
+      {
+        points: "asc",
+      },
```

Comment:
Change this to order by descending. We want to sort by guild members with most points to least points.